packages/@react-aria/tag/src/useTagGroup.ts

```diff
@@ -59,6 +67,12 @@ export function useTagGroup<T>(props: AriaTagGroupProps<T>, state: TagGroupState
       'aria-relevant': 'additions',
       'aria-live': isFocusWithin ? 'polite' : 'off',
       ...focusWithinProps
-    })
+    }),
+    tagGroupActionsProps: gridProps['aria-label'] ? {
+      role: 'group',
+      id: actionsId,
+      'aria-label': stringFormatter.format('actions'),
+      'aria-labelledby': `${gridProps.id} ${actionsId}`
+    } : {}
```

Comment:
Tag group actions feel fairly Spectrum specific to me. Should we move this into RSP?

Reply:
Hmm, yeah I agree. Just moved this logic to the component